app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -211,7 +205,7 @@ class AttendeeFragment : Fragment() {
 
             override fun onItemSelected(p0: AdapterView<*>?, p1: View?, p2: Int, p3: Long) {
                 selectedPaymentOption = paymentOptions[p2]
-                if (selectedPaymentOption == "Stripe")
+                if (selectedPaymentOption == getString(R.string.stripe))
```

Comment:
Not updated

Reply:
This is going to create trouble. Compare with a constant and not a string resource. Use string resource just for presentation